File: public/phase-2-20160620022203/JAVASCRIPT/containing-number.js
// Name : Containing Number
// input:
// n : count of the following numbers
// k : k
// numbers_to_check : 
var idx,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var k = parseInt(elements[1], 10);
var numbers_to_check = new Array([]);
var numbers_to_check_elements = "";
for(idx=0;idx<n;idx++){
	numbers_to_check_elements = readline().split(' ');
	numbers_to_check[idx] = new Array([]);
	for(jdx=0;jdx<1;jdx++){
		numbers_to_check[idx][jdx] = parseInt(numbers_to_check_elements[jdx], 10);
	}
}


// write your code here
// store your results in `result`

// Dummy Data
result = 2;


// output
print(result);

File: public/phase-2-20160620022203/JAVA/containing-number.java
// Name : Containing Number
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : count of the following numbers
		// k : k
		// numbers_to_check : 
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int k = Integer.parseInt(elements[1]);
		int[][] numbers_to_check = new int[n][1];
		String[] numbers_to_check_elements = new String[100];
		for(idx=0;idx<n;idx++){
			numbers_to_check_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<1;jdx++){
				numbers_to_check[idx][jdx] = Integer.parseInt(numbers_to_check_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=2;

		System.out.println(result);

	}
}